build/broccoli/build-packages.js

```diff
@@ -74,7 +74,8 @@ function topsortPackages() {
 function treeForPackage(packagePath) {
   let packageTree = buildPackage({
     projectPath: packagePath,
-    tsconfigPath: TSCONFIG_PATH
+    tsconfigPath: TSCONFIG_PATH,
+    include: ['lib/**/*.ts', 'index.ts']
```

Comment:
@stefanpenner Just a `package.json` file, and `@glimmer/syntax` has a `.js` file but that could be moved somewhere else (it's static and used during the build).